hosted_control_planes/index.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+[id="hcp-overview"]
+= Hosted control planes overview
+include::_attributes/common-attributes.adoc[]
+:context: hcp-overview
+
+You can deploy {product-title} clusters by using two different control plane configurations: standalone or hosted control planes. The standalone configuration uses dedicated virtual machines or physical machines to host the control plane. With hosted control planes for {product-title}, you create control planes as pods on a hosting cluster without the need for dedicated physical machines for each control plane.
```

Comment:
In this line, do dedicated _virtual_ machines need a mention too? I could easily just be misunderstanding though :)
```suggestion
You can deploy {product-title} clusters by using two different control plane configurations: standalone or hosted control planes. The standalone configuration uses dedicated virtual machines or physical machines to host the control plane. With hosted control planes for {product-title}, you create control planes as pods on a hosting cluster without the need for dedicated virtual or physical machines for each control plane.
```

Reply:
Good catch! Yes, it should say "virtual or physical". Fixed ✅